ui/src/components/App/Navigator/Navigator.jsx

```diff
@@ -16,14 +16,9 @@ const Navigator = () => {
           <Nav className="mr-auto">
             <Nav.Link><Link to={'/redcycle'}>Home</Link></Nav.Link>
             <Nav.Link><Link to={'/redcycle/post-creator'}>Post Creator</Link></Nav.Link>
-          </Nav>
-          <Nav>
-            <Nav><a href="http://127.0.0.1:8000/redcycle/api/account/register">Sign Up </a>
-            </Nav>
-            <Nav><a href="http://127.0.0.1:8000/redcycle/api/account/signin">Log in</a>
-            </Nav>
-            <Nav><a href="http://127.0.0.1:8000/redcycle/api/account/logout">Log out</a>
-            </Nav>
+            <Nav.Link href="/redcycle/api/account/register">Sign Up</Nav.Link>
```

Comment:
This [Nav.Link element](https://react-bootstrap.github.io/components/navs/)  has an href property built-in, so I just moved it there. Also, instead of specifying a full domain, you can just specify the path and it will navigate to that path on the current domain